src/main/java/net/snowflake/client/core/auth/oauth/OAuthAccessTokenProviderFactory.java

```diff
@@ -18,10 +18,10 @@
 import net.snowflake.client.log.SFLoggerFactory;
 
 @SnowflakeJdbcInternalApi
-public class AccessTokenProviderFactory {
+public class OAuthAccessTokenProviderFactory {
```

Comment:
Even more precise - `OAuth21`, right?

Reply:
I think this is too detailed. Documentation can point this out but that's about it IMHO.